src/Frontend/src/components/audit/FiltersPanel.vue

```diff
@@ -51,13 +53,31 @@ function findKeyByValue(searchValue: string) {
   }
   return "";
 }
+
+function toggleWildcardToolTip(show: boolean) {
+  if (show) {
+    wildcardTooltipRef.value?.show();
+  } else {
+    wildcardTooltipRef.value?.hide();
+  }
+}
 </script>
 
 <template>
   <div class="filters">
     <div class="filter">
       <div class="filter-label"></div>
-      <div class="filter-component text-search-container"><FilterInput v-model="messageFilterString" placeholder="Search messages..." aria-label="Search messages" /></div>
+      <div class="filter-component text-search-container">
+        <Tippy ref="wildcardTooltipRef" trigger="click" :hideOnClick="false">
+          <template #content>
+            <h4>Use <i class="fa fa-asterisk asterisk" /> to do wildcard searches <i class="fa fa-lightbulb-o" style="color: #e6c201" /></h4>
+            <p>
+              Example: <i><i class="fa fa-asterisk asterisk" />World!</i> or <i>Hello<i class="fa fa-asterisk asterisk" /></i>, to look for <i>Hello World!</i>
+            </p>
+          </template>
+          <FilterInput v-model="messageFilterString" placeholder="Search messages..." aria-label="Search messages" @focus="() => toggleWildcardToolTip(true)" @blur="() => toggleWildcardToolTip(false)" />
```

Comment:
@johnsimons, what do you think about hiding the tooltip when the user starts typing? It isn't very pleasant to always have it visible the entire time the user is focused. 
```suggestion
          <FilterInput v-model="messageFilterString" placeholder="Search messages..." aria-label="Search messages" @focus="() => toggleWildcardToolTip(true)" @blur="() => toggleWildcardToolTip(false)" @input="() => toggleWildcardToolTip(false)" />
```

Reply:
that is a good idea, merged